#id of the request
time request_stamp

#maximal time to wait for new information before the server returns failure to answer the request
std_msgs/Duration max_wait_time